index.md

```diff
@@ -53,25 +61,75 @@ Behandlingsforsikring som dekker følgende (med forbehold om endringer hos Gjens
 - Fysikalsk behandling - 12 behandlinger
 - Online Videolege
 
-### Sykelønn
+#### Sykelønn
 
 Vi har forsikret alle våre arbeidstagere med sjukeforsikring. Denne sikrer at vi ved eventuell sjukemelding utover 14 dager får utbetalt differansen mellom hva NAV utbetaler og hva vi faktisk har i årslønn. NAV utbetaler 6G (599.000). Alt over dette dekkes ikke av NAV. Frem til nå ha 321 tatt kostnaden med differansen. Ettersom vi nå har forsikring, vil ikke 321 lenger ta kostnaden med utbetaling av sjukelønn som ikke dekkes av NAV eller Gjensidige. Vi kan forskuttere dette om dette er ok i følge vilkår hos Gjensidige.
 
 Men dette betyr noe viktig: alle ansatte må fylle ut Helseerklæring innen fristen for å få denne forsikringen. Dette er hver ansatt sin plikt å følge opp, da det ikke enkelt kommer frem på faktura fra Gjensidige. Om du ikke har gyldig helseerklæring vil ikke forsikringen gjelde, da vil 321 verken forskuttere eller dekke differansen mellom NAV sin utbetaling og faktisk årslønn.
```

Comment:
Forenkkles til "Men dette betyr noe viktig: alle ansatte må fylle ut Helseerklæring innen fristen for å få denne forsikringen. Dette er hver ansatt sin plikt å følge opp"? Samt sørge for at dette følges opp ved nyansettelser/onboarding

Reply:
Jeg forenklet denne en god del. La også på denne: Ved eventuell sykdom utbetaler 321 lønn og får refusjon fra NAV og eventuelt forsikring.